core/src/kura.rs

```diff
@@ -734,8 +824,46 @@ mod tests {
         }
     }
 
+    #[test]
+    fn lock_and_unlock() {
+        let dir = tempfile::tempdir().unwrap();
+        {
+            let _store = BlockStore::new(dir.path())
+                .lock()
+                .expect("Lock acquisition failed");
+            fs::File::open(dir.path().join(LOCK_FILE_NAME))
```

Comment:
You can use [`try_exist`](https://doc.rust-lang.org/std/path/struct.Path.html#method.try_exists).

Reply:
Nice, thanks